lighthouse/src/main.rs

```diff
@@ -272,5 +271,5 @@ fn run<E: EthSpec>(
     drop(validator_client);
 
     // Shutdown the environment once all tasks have completed.
-    Ok(environment.shutdown_on_idle())
```

Comment:
This `shutdown_on_idle` call has side-effects and shouldn't be deleted (it could go on the previous line)

Reply:
Oh thanks, seems that I oversaw that, fixed it.